static/src/stylesheets/module/commercial/_creatives.scss

```diff
@@ -730,6 +730,55 @@
 
 }
 
+// Fabric
+// =============================================================================
+
+// Layout
+
+.ad-slot--fabric {
+
+    &.ad-slot--top-banner-ad {
+        width: auto;
+        padding-bottom: 0;
+        padding-left: 0;
+    }
+
+    @include mq(mobile, tablet) {
+        // Full width on mobile fronts
+        width: auto;
+    }
+}
+
+.fabric {
+    height: 250px;
+    transition: height .8s;
+
+    // This is just for demonstration purposes, whilst the ad is populated with ipsum text
+    color: #f0f0f0;
+    background: #303030;
+}
+
+.fabric--expanded {
+    height: 500px;
+}
+
+// Label
+
+.fabric__label-wrapper {
```

Comment:
I guess we could, but is this worth refactoring before we possibly deprecate the old templates anyway?